packages/store-sync/src/internalTableIds.ts

```diff
@@ -0,0 +1,23 @@
+import { resourceIdToHex } from "@latticexyz/common";
+import storeConfig from "@latticexyz/store/mud.config";
+import worldConfig from "@latticexyz/world/mud.config";
+
+// TODO: refactor config to include table IDs (https://github.com/latticexyz/mud/pull/1561)
+
+export const storeTableIds = Object.keys(storeConfig.tables).map((name) =>
+  resourceIdToHex({
+    type: storeConfig.tables[name as keyof typeof storeConfig.tables].offchainOnly ? "offchainTable" : "table",
```

Comment:
`as keyof typeof storeConfig.table` sad that typescript can't infer that from `Object.keys(storeConfig.tables)` 😢

Reply:
agreed! I think lodash has a thing for this, just waiting for them to finish their TS rewrite